rpcs3/Emu/RSX/GL/GLTexture.cpp

```diff
@@ -319,6 +319,14 @@ namespace rsx
 				std::vector<gsl::byte> staging_buffer)
 			{
 				int mip_level = 0;
+				if (is_compressed_format(format))
+				{
+					//Compressed formats have a 4-byte alignment
+					//TODO: Verify that samplers are not affected by the padding
+					width = align(width, 4);
+					height = align(height, 4);
```

Comment:
Ahh. Ok. That's a bug in TextureUtils it seems since you can't have mipmapcount == 0 if w > 0 && h > 0.